src/Components/Layout/StockLayout.css

```diff
@@ -0,0 +1,15 @@
+.layoutHeader {
+    background-color: aqua;
+    height: 12vh;
+}
+
+#ant-layout-sider {
+    background-color: rgb(197, 197, 30);
+    min-width: 400px !important;
```

Comment:
Avoid using important. Use component customization or css chaining

Reply:
Issue is Solved Plx Go throug It